source/ui.js

```diff
@@ -49,6 +49,22 @@ const printCommitLog = async repoUrl => {
 	};
 };
 
+const confirmIgnoredFiles = async pkg => {
+	const ignoredFiles = await util.getNewFilesIgnoredByNpm(pkg);
+	if (ignoredFiles.length === 0) {
+		return {confirm: true};
```

Comment:
Why not simply return `true` here and the answer itself from `inquirer` (instead of returning an object for no reason)?

Reply:
It's returned when prompt is skipped. I wanted to return an object, that has a identical property as the answer from the prompt-module. But i changed to simple `return true`.